syncd/syncd.cpp

```diff
@@ -40,6 +40,7 @@ void exit_and_notify(int status)
     exit(status);
 }
 
+#ifdef BRCMSAI
```

Comment:
Please advise why other vendors need to compile in BRCM specific code?

Denys.

On Tue, Aug 23, 2016 at 9:28 PM, Kamil Cudnik <notifications@github.com>
wrote:

> In syncd/syncd.cpp
> <https://github.com/Azure/sonic-sairedis/pull/64#discussion_r75924156>:
>
> > @@ -40,6 +40,7 @@ void exit_and_notify(int status)
> >      exit(status);
> >  }
> >
> > +#ifdef BRCMSAI
>
> SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE was changed to
> SAI_SWITCH_ATTR_CUSTOM_RANGE_START
>
> simple change could satisfy both: and later on when sai will be stable it
> can be changed back
>
> int32_t id = -1;
>
> #ifndef SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE
> id = SAI_SWITCH_ATTR_CUSTOM_RANGE_BASE;
> #elsif SAI_SWITCH_ATTR_CUSTOM_RANGE_START
> id = SAI_SWITCH_ATTR_CUSTOM_RANGE_START;
>
> #else
> #error invalid atrib custom range not defined
> #endif
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Azure/sonic-sairedis/pull/64/files/941ad4a162b10de7f5ae2dbcc6d89041803af808#r75924156>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABxtcZCOtAOoYsFXfXvmxdf_dEC202tVks5qizu6gaJpZM4JqpD2>
> .
>